### DISCUSSION TITLE
LF APAC 开源布道者分享系列之十：开源布道者的未来 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#25] [Announcements]
# LF APAC 开源布道者分享系列之十：开源布道者的未来 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/what-the-future-of-open-source-advocate/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
Evangelist 在中国其实, 一直有, 
你看各种传销体系中冠销, 或是长期在各种电视购物平台中, 扮演各种专家的布道师...

或是说, 布道这事儿, 在中国很早就发展出完备的操作理论体系: 厚黑学

所以, 感觉, 布道在中国根本不是一个新事物, 而是一个古老到大家习之以常的角色,
但是, 在开源领域独立出一个职位来, 这在中国是全新的期待, 没毛病.

不过, 相比外国普通人都比较轴/单纯, 
中国嘦抗过9年义务教育, 又能自行突围, 主动进入开源领域的,
都不是简单的世界观,

在华布道师, 一般也只能入乡随俗, 遵守基本感染力原则/关键并不在:

- 说什么
- 在哪儿说
- 如何说
- ...
- 而是谁来说...

在代码构造的世界中, 没人能单纯依靠说的东西来积累权威,
打动绝对不傻, 又对世界发展异常理性/偏悲观的中国程序猿,

布道, 真的不能承担 KPI ,而更加象 理想的生活  那种户外真人秀一般,
要全身心的进入这种生活状态, 才可能一言一行, 散发出足以感染其它好人的和善之辉...

对不对?
